frontend/components/feedme/SearchResultsGrid.tsx

```
@@ -0,0 +1,882 @@
+    highlights.forEach(highlight => {
+      const regex = new RegExp(`(${highlight})`, 'gi')
+      result = result.replace(regex, '<mark>$1</mark>')
+    })
+    
+    return result
+  }, [content, highlights, maxLength])
+
+  return (
+    <div 
+      className="text-sm leading-relaxed"
+      dangerouslySetInnerHTML={{ __html: highlightedContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)